UI/js/validations.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Returns a bool depending on whether the value
+ * provided matches an email format or not.
+ */
+const emailValidation = email => {
+  const re = /\S+@\S+\.\S+/;
```

Comment:
```suggestion
  const re = /[a-z0-9-_]+@\w+\.\w{2,4}/;
```
This would be a more appropriate regex for emails, the previous is vulnerable as it matches any non-whitespace character.  otherwise good job

Reply:
Thanks, I will be making the change to the email regex